cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -168,7 +166,9 @@ private void finishInitialization(Bundle savedInstanceState) {
               && savedInstanceState.containsKey(ACTIVE_SHELL_URL_KEY)) {
           shellUrl = savedInstanceState.getString(ACTIVE_SHELL_URL_KEY);
       }
-      mShellManager.launchShell(shellUrl);
+      mShellManager.launchShell("https://www.youtube.com/tv");
```

Comment:
Wouldn't this overwrite the mStartupURL from the Intent in l.131-133? IOW wouldn't this code always force-navigate to YT?

Reply:
Let's move this to finishCobaltInitialization.

Reply:
Only set URL field to gone in this PR, and leave setting URL part unmodified, as this will be implemented with deeplink.